src/ios/CDVWKInAppBrowser.m

```diff
@@ -746,6 +746,9 @@ - (void)createViews
         [self settingForKey:@"AppendUserAgent"] != nil) {
         userAgent = [NSString stringWithFormat:@"%@ %@", userAgent, [self settingForKey:@"AppendUserAgent"]];
     }
+    if (![_browserOptions.appenduseragent isEqualToString:@""]) {
```

Comment:
```suggestion
    if ([_browserOptions.appenduseragent length] > 0) {
```
Perhaps this is clearer and works for the case when string is `nil` as well.

Reply:
@bennordtu Sounds good, lets do that